tests/unit/gaze/gaze_unnest_test.py

```diff
@@ -17,7 +17,7 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
-"""Test GazeDataFrame.unnest()."""
+"""Test Gaze.unnest()."""
```

Comment:
does not match doc strings of other tests, should be

`"""Test Gaze unnest."""

Reply:
good eye